das/util.go

```diff
@@ -12,8 +12,15 @@ import (
 )
 
 func logPut(store string, data []byte, timeout uint64, reader arbstate.DataAvailabilityReader, more ...interface{}) {
-	log.Trace(
-		store, "message", pretty.FirstFewBytes(data), "timeout", time.Unix(int64(timeout), 0),
-		"this", reader, more,
```

Comment:
I think this was meant to be `more...` instead, which would splay the arguments

Reply:
It doesn't work with `more...`

```
go build -o target/bin/daserver "/home/tristan/offchain/nitro/cmd/daserver"
# github.com/offchainlabs/nitro/das
das/util.go:22:22: too many arguments in call to log.Trace
        have (string, string, string, string, time.Time, string, arbstate.DataAvailabilityReader, []interface{})
        want (string, ...interface{})
```

Reply:
Hmm yeah we might've hit a golang edge case. If this new version produces good looking errors LGTM